app/players/player.py

```diff
@@ -0,0 +1,15 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
+        pass
+
+    @abstractmethod
+    def player_info(self) -> None:
```

Comment:
The `player_info` method is annotated to return `None`, but in subclasses, it returns a `str`. You should change the return type annotation to `str` to match the expected return type.